### DISCUSSION TITLE
Improvement to the network app

### DISCUSSION THREAD

**Discussion by peq42** [linuxmint/discussions#639] [Ideas]
It'd be interesting to have a "global settings" to the network app, where settings applied there can be forced as default on every new wifi connection made and/or even current ones(maybe two buttons, one for each?)

I suggest this because, if installed on a laptop, a single person can connect to dozens or even hundreds of different wifi connections, which makes it a MASSIVE hassle to manually change every a setting in single one.

I'd also love a setting to change the default DNS IP on all my connections. I tried changing it on my laptop(has the latest version of Linux Mint Cinnamon) and honestly couldn't properly do it, as I was required to meddle with commands in the terminal

**Comment by claudiux** [comment]
DNS IPs are generally supplied by your connection's DHCP service. Overwriting them can be seen as an open door to a security problem. If you want to do this, use your own script or create your own applet, but it doesn't seem like a good idea to provide it through Cinnamon's network settings.

**Reply by peq42** [reply]
> DNS IPs are generally supplied by your connection's DHCP service. Overwriting them can be seen as an open door to a security problem. If you want to do this, use your own script or create your own applet, but it doesn't seem like a good idea to provide it through Cinnamon's network settings.

basically every single article you can find will recommend users to change their DNS from the default offered by their ISP, usually to something like cloudflare or google's.

**Reply by claudiux** [reply]
> basically every single article you can find will recommend users to change their DNS from the default offered by their ISP, usually to something like cloudflare or google's.

If these articles are written by the same people who advise you to put your system and personal data on the same partition, I'm suspicious. These articles recommend using the DNS of the companies that pay them.

However, you can install a DNS service on your computer with bind9 ( find a good DNS-howto) and use it for all your connections.
